src/components/HomePage.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { INCREMENT, DECREMENT, ADD_COUNTER } from '../redux/actionTypes';
+
+function HomePage({
+  counters, addCounter, increment, decrement,
+}) {
+  if (counters && addCounter) {
+    return (
```

Comment:
It is better to decrease level of nesting:
```js
if (!counters || !addCounter) {
  return null;
}
```

And you should always return something.